src/main/java/codeu/model/data/User.java

```diff
@@ -39,6 +40,7 @@ public User(UUID id, String name, String passwordHash, Instant creation, Boolean
     this.passwordHash = passwordHash;
     this.creation = creation;
     this.isAdmin = isAdmin;
+    this.bio = "";
```

Comment:
I did not overload the constructor cause I did not find the bio to be a field necessary for the construction of the user object. Instead, I added a setter to allow a user to update their bio.